docs/developer-guide/code-guidelines.rst

```diff
@@ -2,7 +2,7 @@
 Code Guidelines
 ***************
 
-Coding should follow the CTA coding guidelines from the **CTA Code
+Coding should follow the CTA coding guidelines from the **CTAO Code
```

Comment:
This could use the correct title and provide a reference. 

```
CTAO Software Programming Standards :cite:p:`bib-entry`
```

Reply:
Is it public? I guess we can add a link to the private version in any case

Reply:
No, but we have tons of other documents in the bibliography with the link to the computing-info redmine dms